docs/stage/spider-data-catalog.md

```diff
@@ -35,3 +38,29 @@ Run the SQL:
     DELETE FROM publisher_feed;
     DELETE FROM publisher;
     DELETE FROM spider_data_catalog_error;
+
+## To run the system with only some publishers / feeds
+
+To debug a problem that occurs with a specific publisher's data, you may want to get data for that publisher only.
+
+### Delete entries 
```

Comment:
I'm not sure it's clear that these are alternatives. Maybe put something in the section above to say that there are two ways?